src/appengine/libs/issue_management/issue_tracker_utils.py

```diff
@@ -154,10 +156,17 @@ def get_similar_issues_url(issue_tracker, testcase, only_open=True):
 def get_issue_url(testcase):
   """Return issue url for a testcase."""
```

Comment:
Maybe expand on description here that this is for display and account for group bug.

Reply:
Done!